packages/experimental/Shadow/src/Shadow.tsx

```diff
@@ -1,31 +1,25 @@
 import * as React from 'react';
 import { View } from 'react-native';
-import { ShadowProps } from './Shadow.types';
+import { ShadowProps, ShadowTokens, shadowName } from './Shadow.types';
+import { compressible, buildUseTokens, useFluentTheme } from '@fluentui-react-native/framework';
+import { shadowStyleFromTheme } from './shadowStyle';
 
-export const Shadow: React.FunctionComponent<ShadowProps> = (props: ShadowProps) => {
-  const noOffset = { width: 0, height: 0 };
-  const neutralShadowAmbient = { shadowOffset: noOffset, shadowOpacity: 0.12, shadowRadius: 2 };
-  const neutralShadowAmbientDarker = { shadowOffset: noOffset, shadowOpacity: 0.2, shadowRadius: 8 };
-  const ambientShadow = {
-    '2': neutralShadowAmbient,
-    '4': neutralShadowAmbient,
-    '8': neutralShadowAmbient,
-    '16': neutralShadowAmbient,
-    '28': neutralShadowAmbientDarker,
-    '64': neutralShadowAmbientDarker,
-  };
-  const keyShadow = {
-    '2': { shadowOffset: { width: 0, height: 1 }, shadowOpacity: 0.14, shadowRadius: 2 },
-    '4': { shadowOffset: { width: 0, height: 2 }, shadowOpacity: 0.14, shadowRadius: 4 },
-    '8': { shadowOffset: { width: 0, height: 4 }, shadowOpacity: 0.14, shadowRadius: 8 },
-    '16': { shadowOffset: { width: 0, height: 8 }, shadowOpacity: 0.14, shadowRadius: 16 },
-    '28': { shadowOffset: { width: 0, height: 14 }, shadowOpacity: 0.24, shadowRadius: 28 },
-    '64': { shadowOffset: { width: 0, height: 32 }, shadowOpacity: 0.24, shadowRadius: 64 },
+const useShadowTokens = buildUseTokens<ShadowTokens>(() => ({}), shadowName);
+
+export const Shadow = compressible<ShadowProps, ShadowTokens>((props: ShadowProps) => {
+  const { depth } = props;
+  const theme = useFluentTheme();
+
+  const shadowStyle = shadowStyleFromTheme(theme, 'shadow' + depth.toString());
+
+  return (extra: ShadowProps, children: React.ReactNode) => {
+    return (
+      <View style={shadowStyle.ambient}>
+        <View style={shadowStyle.key}>{children}</View>
+      </View>
```

Comment:
I suggest we pass the rest of props down to the inner view. That way, the client can use ShadowComponent like any other View, with ViewProps, but also gets a shadow effect. 

The convention of passing unrelated props is listed / documented here, but I find it not very readable: https://reactjs.org/docs/higher-order-components.html#convention-pass-unrelated-props-through-to-the-wrapped-component

```
const mergedProps = mergeProps(
  {style: {shadowStyle.key} },
  ...rest
 );
 
 
 return (
  <View style={shadowStyle.ambient}>
         <View {...mergedProps}>{children}</View>
  </View>
 )
 ```